pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeprojectmetadata/create.yaml

```diff
@@ -16,7 +16,7 @@ apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeProjectMetadata
 metadata:
   annotations:
-    cnrm.cloud.google.com/project-id: metadata-${uniqueId}
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
```

Comment:
Just when I thought I was getting the hang of it, a non-IAP example appears!  Can you add a comment as to why here?